canonical/snapcraft.io
# Add model policies view

## Done
Built the policies view

## How to QA
- Go to https://snapcraft-io-4346.demos.haus/login-beta and login
- Go to https://snapcraft-io-4346.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models/test_model_without_api_key/policies
- Check that there are policies in the table
- Check that the signing key links to the signing key page (this will be blank)
- Check that you can create a policy

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-4738
Fixes https://warthogs.atlassian.net/browse/WD-4740
